skel/rollup.config.js

```diff
@@ -10,7 +10,7 @@ const umd = meta.umd || name;
 
 export default [
   {
-    input: 'src/main.js',
+    input: 'index.js',
```

Comment:
again, this isn't obviously part of a netstring feature, so some mention of it in the commit message seems worthwhile.